src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryCustomObject.cs

```diff
@@ -15,13 +15,39 @@ internal static partial class Library
     {
         internal class JsonCustomObject : ICustomObject
```

Comment:
>JsonCustomObject [](http://example.com/codeflow?start=23&length=16)

nit: can you move this to its own file? It's very specific to json. (in the future, we might even consider moving it to its own assembly along with ParseJson and our STJ dependency)